middleware.js

```diff
@@ -31,6 +31,30 @@ const vueMiddleware = (options = defaultOptions) => {
     res.end(source)
   }
 
+  function injectSourceMapsToScript (script) {
+    const map = Base64.toBase64(
+      JSON.stringify(script.map)
+    )
+
+    return {
+      ...script,
+      code: `//# sourceMappingURL=data:application/json;base64,${map}\n` + script.code
+    }
+  }
+
+  function injectSourceMapsToStyles (styles) {
+    return styles.map(s => {
```

Comment:
Isn't the logic duplicated with `injectSourceMapsToScript` here?

Reply:
Can just rename `injectSourceMapsToScript` and reuse it here

Reply:
@yyx990803 JS comment is `//`, CSS one is `/* */`

Reply:
but maybe CSS-style will work in JS too :thinking:

Reply:
no, it doesn't, but we can add something like `commStyle` parameter to our fn.

Reply:
@yyx990803 I've done some refactor, please review